pkg/storaged/utils.js

```diff
@@ -132,6 +132,26 @@
         }
     };
 
+    utils.validate_fsys_label = function validate_fsys_label(label, type) {
+        var fs_label_max = {
+            "xfs":   12,
+            "ext4":  16,
+            "vfat":  11,
+            "ntfs": 128,
+        };
```

Comment:
I originally thought [udisks](https://github.com/storaged-project/udisks/blob/master/src/udiskslinuxfsinfo.c) would have a list of these label lengths, but it doesn't. So we even couldn't re-use some validation function from it.

Reply:
Yes, exactly.